src/containers/DebitCard/bridgeCard/fundCard.tsx

```
@@ -1072,14 +1457,15 @@ export default function BridgeFundCardScreen({ route }: { route: any }) {
               {'~' +
                 (isCrpytoInput
                   ? (!isNaN(parseFloat(usdAmount))
-                      ? formatAmount(usdAmount).toString()
+                      ? formatAmount(usdAmount)
                       : '0.00') + ' USD'
                   : (!isNaN(parseFloat(cryptoAmount))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)